### PR TITLE
Register GeneralizedSampling: v0.0.1, v0.0.2, v0.0.3, v0.0.4

### PR DESCRIPTION
**Pull request by robertdj**

### CODE REVIEW

File: GeneralizedSampling/url [hunk 1]
```diff
@@ -0,0 +1 @@
+github:robertdj/GeneralizedSampling.jl.git
```
Comment (tkelman) [inline GeneralizedSampling/url]: this should be `https://github.com/robertdj/GeneralizedSampling.jl.git`

**Comment by dpsanders** [comment]
I think the name is too generic -- I assumed from the name that this was about some kind of Monte Carlo or statistics work.

File: GeneralizedSampling/url (hunk 1 above)
Comment (robertdj) [inline GeneralizedSampling/url]: Argh... Old habit with `.ssh/config`. Is it correct that this is not in the package, but in `METADATA`? So I shouldn't change anything in the repository, but in `METADATA/GeneralizedSampling`?

Comment (tkelman) [inline GeneralizedSampling/url]: correct

**Comment by robertdj** [comment]
Well, the theory implemented is called *generalized sampling*...

File: GeneralizedSampling/url (hunk 1 above)
Comment (robertdj) [inline GeneralizedSampling/url]: Cool. The commit is from another computer, so it wont be until tomorrow. After changing the offending line, should I manually `git push` the `metadata-v2` branch or use `Pkg.publish` in Julia?

Comment (tkelman) [inline GeneralizedSampling/url]: For this kind of one-liner change, you can make the modification to this branch of your metadata fork from the github web interface - go to https://github.com/robertdj/METADATA.jl/blob/pull-request/bd952480/GeneralizedSampling/url and click the "edit" pencil icon. For any larger changes it cuts down on noise a bit if you manually make the commit to the same `pull-request/bd952480` branch of your fork and push it to there, the new commit will show up in this same PR.

Comment (robertdj) [inline GeneralizedSampling/url]: You really know how to handle Git(Hub)! It's changed now.

File: GeneralizedSampling/versions/0.0.1/sha1 [hunk 2]
```diff
@@ -0,0 +1 @@
+3ac71e76ae8daaedfa14d9d0b12d3401afe06487
```
Comment (tkelman) [inline GeneralizedSampling/versions/0.0.1/sha1]: You should probably delete the 0.0.1, 0.0.2, and 0.0.3 sha1 files unless you expect them to work on Julia 0.3 and earlier.

Comment (robertdj) [inline GeneralizedSampling/versions/0.0.1/sha1]: Okay. Should I then also delete the tags in the package repo? And make the current 0.0.4 into 0.0.1?

Comment (tkelman) [inline GeneralizedSampling/versions/0.0.1/sha1]: that's not needed, just deleting the sha files here and saying 0.0.4 is the first released tag would probably be easiest

Comment (robertdj) [inline GeneralizedSampling/versions/0.0.1/sha1]: What possibly future releases? Wont the 0.0.1, 0.0.2 & 0.0.3 be included there?

Comment (tkelman) [inline GeneralizedSampling/versions/0.0.1/sha1]: don't think so, I believe the next time you do Pkg.tag it should use 0.0.5  as the next patch by default, or 0.1.0 if you do `Pkg.tag("GeneralizedSampling", :minor)`

**Comment by tkelman** [comment]
Bump, @robertdj could you delete the 3 sha1 files for the early versions that don't have corresponding `requires` files? You can do this through the web interface if that's easiest, by clicking the trash icon at https://github.com/robertdj/METADATA.jl/blob/pull-request/bd952480/GeneralizedSampling/versions/0.0.1/sha1, https://github.com/robertdj/METADATA.jl/blob/pull-request/bd952480/GeneralizedSampling/versions/0.0.2/sha1, and https://github.com/robertdj/METADATA.jl/blob/pull-request/bd952480/GeneralizedSampling/versions/0.0.3/sha1

**Comment by robertdj** [comment]
Yes, I'll get on it. Vacation got in the way.

**Pull request merged**